ci-operator/step-registry/gather/core-dump/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
```

Comment:
@astoycos you wrote this step, so add yourself to the OWNERS, and maybe tim and some others too. Clayton, wking, abhinav, at least probably don't need (or want) to be on here.

Reply:
Sounds good